model-optimizer/extensions/front/mxnet/modulated_deformable_conv_ext.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from mo.front.extractor import FrontExtractorOp
+from mo.front.mxnet.extractors.utils import get_mxnet_layer_attrs
+from mo.ops.deformable_convolution import DeformableConvolution
+
+
+class ModulatedDeformableConvolutionExtractor(FrontExtractorOp):
+    op = '_contrib_ModulatedDeformableConvolution'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        attr = get_mxnet_layer_attrs(node.symbol_dict)
+
+        kernel = attr.tuple("kernel", int, None)
+        stride = attr.tuple("stride", int, tuple(np.ones(len(kernel), dtype=np.int64)))
+        padding = attr.tuple("pad", int, tuple(np.zeros(len(kernel), dtype=np.int64)))
+        dilate = attr.tuple("dilate", int, tuple(np.ones(len(kernel), dtype=np.int64)))
```

Comment:
Please fix specification because I see that default value for all these attributes is `None`.